zebrad/tests/common/lightwalletd/send_transaction_test.rs

```diff
@@ -176,8 +176,13 @@ pub async fn run() -> Result<()> {
         assert_eq!(response, expected_response);
     }
 
-    tracing::info!("waiting for mempool to verify some transactions...");
-    zebrad.expect_stdout_line_matches("sending mempool transaction broadcast")?;
+    // The timing of verification logs are unrealiable, so we've disabled this check for now.
```

Comment:
Not at all important so I'll update this in the following PR

```suggestion
    // The timing of verification logs are unreliable, so we've disabled this check for now.
```